src/assets/stylesheets/scene-ui.scss

```diff
@@ -81,15 +83,15 @@
 
 :local(.attribution) {
   font-size: 1.0em;
-  white-space: wrap;
+  white-space: normal;
 
   .remix {
     font-size: 0.8em;
   }
 
   a {
     font-size: 0.8em;
-    color: black;
+    color: theme.$text1-color;
```

Comment:
Not sure about the target style for this page but we have moved from a dark font to a light one. I find it harder to read the text.

Reply:
Especially true against light-colored scene previews for dark and darker previews for light, also it's not very clear to me that this is a link to the glb. Will make notes here: [Design Link](https://www.figma.com/file/J6YKkb2vWDlk81zm4kxUzA/Hubs-discoverability-and-onboarding?node-id=903%3A1324)

https://user-images.githubusercontent.com/4493657/147691074-dd975838-39f5-4e72-867e-6d1effc8ed23.mp4

Reply:
Thanks for the Figma link! that's what I was looking for. Makes sense.